{"uuid": "cbc50b67-fec3-49fa-9971-08c4136a6897", "children": ["ca2f66d8-9f08-454b-87f0-ba4a28ecb755"], "befores": [{"name": "_Class__pytest_setup_class", "status": "broken", "statusDetails": {"message": "TypeError: __init__() missing 1 required positional argument: 'dev'\n", "trace": "  File \"/Users/chenjinfei/opt/anaconda3/lib/python3.8/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/chenjinfei/opt/anaconda3/lib/python3.8/site-packages/_pytest/fixtures.py\", line 964, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/chenjinfei/opt/anaconda3/lib/python3.8/site-packages/_pytest/fixtures.py\", line 788, in call_fixture_func\n    res = next(it)\n  File \"/Users/chenjinfei/opt/anaconda3/lib/python3.8/site-packages/_pytest/python.py\", line 701, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"/Users/chenjinfei/opt/anaconda3/lib/python3.8/site-packages/_pytest/python.py\", line 632, in _call_with_optional_argument\n    func(arg)\n  File \"/Users/chenjinfei/project/pythonProject/mtxweb/case/test_login.py\", line 10, in setup_class\n    self.logobj = Mtx_Login()\n  File \"/Users/chenjinfei/project/pythonProject/mtxweb/page/Mtx_Login.py\", line 11, in __init__\n    super().__init__()\n"}, "start": 1615772219047, "stop": 1615772219047}], "start": 1615772219047, "stop": 1615772219050}